packages/admin/admin-date-time/src/timePicker/FinalFormTimePicker.tsx

```diff
@@ -2,7 +2,7 @@ import { FieldRenderProps } from "react-final-form";
 
 import { TimePicker, TimePickerProps } from "./TimePicker";
 
-export type FinalFormTimePickerProps = TimePickerProps & FieldRenderProps<string, HTMLInputElement | HTMLTextAreaElement>;
+type FinalFormTimePickerProps = TimePickerProps & FieldRenderProps<string, HTMLInputElement | HTMLTextAreaElement>;
```

Comment:
@jamesricky is it okay to remove this export? Does this cause problems with theming?